### ISSUE TITLE
React Error in Dev mode

### ISSUE THREAD

**Issue by Slashgear** [paularmstrong/react-component-benchmark#2]
# Problem
When I try to use `react-component-benchmark` on my styleguide app locally in dev mode, I get this error:

```
Uncaught Invariant Violation: Maximum update depth exceeded. 
This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. 
React limits the number of nested updates to prevent infinite loops.
```

This is completely *Ok* because React want to prevent infinite render loop in dev mode.

Have you encountered this kind of problem?

Do you have any idea how to bypass it?

I really want to be able to use it in dev in order to see if my local changes affects rendering performances.

I already tried to use an error boundary. 😞

**Comment by paularmstrong** [comment]
I think you need to reduce the number of cycles you're running. 

Long-term, "correct" fix, would require some sort of pause between updates.

**Comment by paularmstrong** [comment]
This will be fixed in #8, thanks to the ability to delay re-renders to separate ticks.
